diff --git a/dist/metro/index.js b/dist/metro/index.js
--- a/dist/metro/index.js
+++ b/dist/metro/index.js
@@ -177,17 +177,12 @@
                 : getNativeJS((0, css_to_rn_1.cssToReactNativeRuntime)(css, options), debug)));
             debug(`virtualStyles.emit ${platform}`);
             haste.emit("change", {
-                eventsQueue: [
-                    {
-                        filePath,
-                        metadata: {
-                            modifiedTime: Date.now(),
-                            size: 1,
-                            type: "virtual",
-                        },
-                        type: "change",
-                    },
-                ],
+                changes: {
+                    addedFiles: [],
+                    modifiedFiles: [],
+                    removedFiles: [],
+                },
+                rootDir: "",
             });
         }).then((css) => {
             debug(`virtualStyles.initial ${platform}`);
